fitsio/fitslib.py

```diff
@@ -1773,7 +1781,11 @@ def insert_column(self, name, data, colnum=None):
         if name in self._colnames:
             raise ValueError("column '%s' already exists" % name)
 
-        descr=data.dtype.descr
+        if IS_PY3 and data.dtype.char == 'U':
+            descr = data.astype('S', copy=False).dtype.descr
```

Comment:
It might be better to adjust the descriptor here instead of casting the whole array.